oura-library/src/main/kotlin/org/radarbase/oura/converter/OuraPersonalInfoConverter.kt

```diff
@@ -0,0 +1,49 @@
+package org.radarbase.oura.converter
+
+import com.fasterxml.jackson.databind.JsonNode
+import org.radarcns.connector.oura.OuraPersonalInfo
+import org.slf4j.LoggerFactory
+import java.time.Instant
+import java.time.OffsetDateTime
+import org.radarbase.oura.user.User
+
+class OuraPersonalInfoConverter(
+    private val topic: String = "connect_oura_personal_info",
+) : OuraDataConverter {
+    override fun processRecords(
+        root: JsonNode,
+        user: User
+    ): Sequence<Result<TopicData>> {
+        val array = root.optArray("data")
```

Comment:
Personal info data is not under `data` property since there aren't multiple results.